File: API/Farmers/Models/CollectionDetails.cs
namespace Farmers.Models

{
    public class CollectionDetails
    {
        public int Id { get; set; }
        public int FarmerId { get; set; }
        public string? CropName { get; set; }
        public string? ContainerType { get; set; }
        public string? Grade { get; set; }
        public int Quantity { get; set; }
        public int ManagerId { get; set; }
        public double TotalWeight { get; set; }
        public double NetWeight { get; set; }
        public DateTime CollectionDate { get; set; }
    }
}

File: UI/EAgroUIApp/src/app/Models/collection-details.ts
export class CollectionDetail {
    constructor(
        public id: number,
        public farmerId: number,
        public farmerName: string,
        public cropName: string,
        public containerType: string,
        public grade: string,
        public quantity: number,
        public inspectorId: number,
        public inspectorName: string,
        public totalWeight: number,
        public netWeight: number,
        public collectionDate: string
    ) { }
}
